### PR TITLE
Extracted IDL interfaces and related algorithms to API section

### PR DESCRIPTION
**Pull request by avayvod**
Basically wrapped everything in a new section with <h2> title API, changed internal <h2> to <h3> and indented everything by one tab.

### CODE REVIEW

**Comment by avayvod** [comment]
@mounirlamouri PTaL after the connect() PR

File: index.html [hunk 1]
```diff
@@ -293,416 +293,419 @@
...
         <p>
-          Some <a>remote playback devices</a> may only be able to display a
-          subset of Web content because of functional, security or hardware
-          limitations. Examples are set-top boxes, smart TVs or networked
-          speakers capable of rendering only certain formats of video and/or audio.
-          We say that such a device is a <dfn>compatible remote playback device</dfn> for a
-          <a>media element source</a> if the <a>user agent</a> can
-          reasonably guarantee that the remote playback of the media speficied by the source
-          on that device will succeed.
+          <b>Draft summary:</b><br>
```
Comment (mounirlamouri) [inline index.html:315]: Do we still need that? Isn't getAvailability defined now?

**Comment by mounirlamouri** [comment]
lgtm assuming you only moved stuff

File: index.html (hunk 1 above)
Comment (avayvod) [inline index.html:315]: Good point, removed.

**Pull request merged**